resolves browser back button issue on login route when user is not logged in

anu-007 opened the pull request:
<!-- 
(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
-->

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes <!-- use `ember test` to run all the tests -->
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
resolves browser back button issue on login route when user is not logged in

#### Changes proposed in this pull request:

-changed anchor tag with link-to helper
-added nav-bar.scss file and moved inline styles to this file
-moved inline styles to welcome.scss file

demo- https://open-event-frontend-branch-2.herokuapp.com/
<!-- Add the issue number that is fixed by this PR (In the form Fixes #123) -->

Fixes #365

CosmicCoder96 commented on app/styles/partials/nav-bar.scss:
```diff
@@ -0,0 +1,10 @@
+.item > .ui.blue.button {
```
this will apply to any generic item and blue button, Please wrap it so it is specific to the navbar

anu-007 replied:
@CosmicCoder96 done, please review.

CosmicCoder96 commented on app/styles/partials/welcome.scss:
```diff
@@ -21,6 +21,8 @@
     .button {
       margin-top: 20px;
       font-size: 16px;
+      text-transform: uppercase;
```
this will apply to any generic item and blue button, Please wrap it so it is specific to the navbar

niranjan94 commented on app/styles/partials/nav-bar.scss:
```diff
@@ -0,0 +1,10 @@
+
+@media screen and (max-width: 768px) {
+  .right.menu > .item > .ui.blue.button {
+    button-text: small text;
```
what is `button-text` ?

anu-007 replied:
it is previously present in the anchor tag and I blindly put this in this file but I checked it now it doesn't do anything so sorry, removing it.

CosmicCoder96 commented on app/styles/partials/nav-bar.scss:
```diff
@@ -0,0 +1,4 @@
+.right.menu > .item > .ui.blue.button {
```
@anu-007 `right menu` is again a generic semantic UI class https://semantic-ui.com/collections/menu.html
This is not what I meant by wrapping. I meant to say that make it specific to this navbar, right menu may be used some place else too, please change this.

geekyd replied:
@anu-007 Please use something like 
```
.nav-bar > .item > .ui.blue.button {
}
```
Add this `nav-bar` class to the div containing `right menu` class.

anu-007 replied:
@CosmicCoder96 I misunderstood first-time @geekyd thanks for explaining.done changes please review.thanks

anu-007 commented:
@CosmicCoder96 it is already done for mobile navbar please check.

CosmicCoder96 commented:
@anu-007 Yes, I forgot even though they have different component files, they share the create event button 😅

anu-007 commented:
@CosmicCoder96 I just transferred inline CSS to the .scss file nothing else, I also checked for overflow related problems couldn't find any.

anu-007 commented:
@CosmicCoder96 I looked into this issue and found that I missed some class.now I have added them thanks for pointing that out also I have corrected some condition checks like in `mobile only row` there is no need to check if the device is mobile, corrected them too.

CosmicCoder96 commented on app/templates/components/nav-bar.hbs:
```diff
@@ -47,32 +47,32 @@
       {{/if}}
       <div class="item">
-        <a class="ui blue button button-text {{if device.isMobile 'small text'}}" style="text-transform: uppercase; font-weight: 400;" href="{{href-to 'create'}}">
+        {{#link-to 'create' class='ui blue button button-text small text'}}
```
@anu-007 Didn't you remove the `button-text` class earlier?

anu-007 replied:
@CosmicCoder96 yes you are right I removed that earlier and it was a very silly mistake by me I will be more careful in future.thanks!!

geekyd commented:
@niranjan94 Please approve & merge this. Thanks
